packages/apps/transfer/next.config.js

```diff
@@ -3,14 +3,17 @@ const nextTranslate = require('next-translate-plugin');
 
 module.exports = nextTranslate({
   reactStrictMode: true,
-  pageExtensions: ['^((?!(test|spec)).)*.tsx$'],
-  webpack: (config, { dev }) => {
+  pageExtensions: process.env.NODE_ENV === "production" ? ['(?<!(spec|test).)tsx'] : ['tsx'],
+  webpack: (config, { isServer }) => {
     config.module.rules.push(
       {
         test: /\.(spec|test).*$/,
         loader: 'ignore-loader'
       }
     );
+    if (!isServer) {
+      config.resolve.fallback.fs = false;
```

Comment:
Perhaps add a comment here indicating why this is done? 😁

Reply:
this is regarding the erorr of "module fs not found" cause by default it'll put fs codes on client (if we use fs in components) which will be executed on server, but not the client, so this is for later issues that might happen by using fs module in app directory